src/libs/PolicyUtils.ts

```diff
@@ -763,6 +763,13 @@ function hasDependentTags(policy: OnyxEntry<Policy>, policyTagList: OnyxEntry<Po
     return Object.values(policyTagList ?? {}).some((tagList) => Object.values(tagList.tags).some((tag) => !!tag.rules?.parentTagsFilter || !!tag.parentTagsFilter));
 }
 
+function hasIndependentTags(policy: OnyxEntry<Policy>, policyTagList: OnyxEntry<PolicyTagLists>) {
+    if (!policy?.hasMultipleTagLists) {
+        return false;
+    }
+    return Object.values(policyTagList ?? {}).every((tagList) => Object.values(tagList.tags).every((tag) => !tag.rules?.parentTagsFilter && !tag.parentTagsFilter));
```

Comment:
@allgandalf I am getting this error on all my existing sessions. Once I am logout and login again it goes away. Best to add `tagList.tags ?? {}`

<img width="1210" alt="desktop-error-tags" src="https://github.com/user-attachments/assets/4e2e3cef-fba9-4ad5-a667-7f2742c21e4b" />

<img width="426" alt="mweb-chrome-error-tags" src="https://github.com/user-attachments/assets/d8e40a5a-9bda-47cb-a9bd-1503cda63ef5" />

Reply:
@allgandalf Were you able to follow-up on this?